CODE-OF-CONDUCT.md

```diff
@@ -0,0 +1,134 @@
+# Solo Community Code of Conduct
+
+All members of the Solo community must abide by the following Code of Conduct,
+adapted from the [Contributor Covenant][homepage]. Only by respecting each
+other can we develop a productive, collaborative community.
+
+## Our Pledge
+
+As members, contributors, and leaders of the Solo community, we pledge to make
+participation in our community a harassment-free experience for everyone,
+regardless of age, body size, visible or invisible disability, ethnicity, sex
+characteristics, gender identity and expression, level of experience,
+education, socio-economic status, nationality, personal appearance, race,
+religion, or sexual identity and orientation.
+
+We pledge to act and interact in ways that contribute to an open, welcoming,
+diverse, inclusive, and healthy community.
+
+## Our Standards
+
+Examples of behavior that contributes to a positive environment for our
+community include:
+
+* Demonstrating empathy and kindness toward other people
+* Being respectful of differing opinions, viewpoints, and experiences
+* Giving and gracefully accepting constructive feedback
+* Accepting responsibility and apologizing to those affected by our mistakes,
+  and learning from the experience
+* Focusing on what is best not just for us as individuals, but for the
+  overall community
+* Practicing [Slack etiquette](https://slack.com/blog/collaboration/etiquette-tips-in-slack) when interacting with community members via Slack
+
+Examples of unacceptable behavior include:
+
+* Post non relevant spam contents
```

Comment:
nit: Unsure what Relevant spam contents would be.
Therefore I propose something more like the following (though most likely not exactly the following)
```suggestion
* Posting content that is non-relevant, unhelpful, or solicitous (spam)
```